apiserver/cmd/apiserver/apiserver.go

```diff
@@ -36,14 +38,24 @@ func main() {
 	// Make sure the RNG is seeded.
 	seedrng.EnsureSeeded()
 
+	// The ConsistentListFromCache feature gate requires our resourceStore
+	// to support method RequestWatchProgress, which it does not.  Force-disable
+	// the gate.
+	err := feature.DefaultMutableFeatureGate.SetFromMap(map[string]bool{string(features.ConsistentListFromCache): false})
+	if err != nil {
+		klog.Errorf("Error setting feature gates.")
+		logs.FlushLogs()
```

Comment:
Ah yep, probably needs to be.  I think I thought "this should go as early as possible in case any initialisation depends on the flags" but we can't really avoid initialising the logger first.